### DISCUSSION TITLE
Vowifi problem

### DISCUSSION THREAD

**Discussion by Nelos88** [kyujin-cho/pixel-volte-patch#90] [Q&A]
Hello.after the march update volte came finally official with the update but vowifi no so i tried multiple times to activate vowifi with pixel IMS but it doesn't works at all even though it's activated at pixel IMS settings and phone settings.before march update both volte and vowifi worked perfect with pixel IMS. My phone is pixel 7 pro and my carrier is COSMOTE from Greece. Any solution please ? @kyujin-cho

**Comment by Puddiiing** [comment]
Use ver 1.2.2
1.2.3 is broken somehow🤗

**Reply by Nelos88** [reply]
I tried it already without success.

**Reply by Puddiiing** [reply]
Reset network setting and use ver 1.2.1🤗
